.github/CODEOWNERS

```diff
@@ -9,5 +9,6 @@
 # explicitly taken by someone else.
 *                               @googleapis/yoshi-ruby
 
+/cloud-sql/    @GoogleCloudPlatform/infra-db-dpes
```

Comment:
Please also include `@googleapis/yoshi-ruby` for all directories.

Reply:
Github doesn't support cross-org teams in CODEOWNERs. The team has to have explicit write access, which can't be given to a team outside the org. None of the examples of `@googleapis/yoshi-ruby` are currently effective because of this. 

If you have an equivalent @GoogleCloudPlatform team I can add them in instead, but otherwise it won't work.

Reply:
Oh, sorry about that, I didn't look close enough. Yes, it should be `@GoogleCloudPlatform/yoshi-ruby`. In fact, would you mind fixing the other instances of yoshi-ruby in this file as well?